ocsp/responder/redis/redis_source.go

```diff
@@ -82,7 +96,11 @@ func (src *redisSource) Response(ctx context.Context, req *ocsp.Request) (*respo
 
 	if src.isStale(resp) {
 		src.counter.WithLabelValues("stale").Inc()
-		return src.signAndSave(ctx, req, "stale_redis")
+		freshResp, err := src.signAndSave(ctx, req, "stale_redis")
+		if err != nil {
+			return &responder.Response{Response: resp, Raw: respBytes}, nil
```

Comment:
Seems like we should probably log or increment a metric, so that if these life-signings start to fail we can be notified of it. Or is the idea just that we'd alert on "proportion of responses served Stale is too high"?

Reply:
We get the ocsp_response_ages histogram from `*ocsp.Responder`, which wraps this source. I've checked and we do have an alert on that.

We get metrics for signing failures from the grpc_ecosystem interceptor - they would show up as errors on the GenerateOCSP RPC. We don't have a specific alert on that, but I think that's reasonable - it's a cause, not a symptom; the symptom would be that we start serving stale responses.